addon/TabTracker.js

```diff
@@ -43,6 +40,12 @@ TabTracker.prototype = {
     return this._tabData;
   },
 
+  init(trackableURLs, placesQueries, experimentId) {
```

Comment:
Yup I was thinking in the case for tiles we won't have both ```trackableURLs``` or ```placesQueries``` so we'll have to have a default for those. I'll file the ticket